scripts/src/public/services/orders.ts

```diff
@@ -370,23 +375,25 @@ export async function cancelOrder(orderId: string): Promise<void> {
 }
 
 export async function getOrderHistory(
-	userId: string,
+	user: User,
+	deviceId: string,
 	filters: { origin?: db.OrderOrigin; offerId?: string; },
 	limit: number = 25,
 	before?: string,
 	after?: string): Promise<OrderList> {
 
 	// XXX use the cursor input values
 	const status: db.OrderStatusAndNegation = "!opened";
+	const wallet = (await user.getWallets(deviceId)).lastUsed();
```

Comment:
shouldn't `getWallets(deviceId)` only return a single wallet?  Why don't you create `getWallet(deviceId)` which is separate from the `getWallets()` function?

Reply:
a device might have more than one wallet